libstuff/SHTTPSManager.cpp

```diff
@@ -37,6 +37,23 @@ void SHTTPSManager::closeTransaction(Transaction* transaction) {
     delete transaction;
 }
 
+int SHTTPSManager::getHTTPResponseCode(const string& methodLine) {
+    // This code looks for the first space in the methodLine, and then for the first non-space
+    // after that, and *then* parses the response code. If we fail to find such a code, or can't parse it as an
+    // integer, we default to 400.
+    size_t offset = methodLine.find_first_of(' ', 0);
+    offset = methodLine.find_first_not_of(' ', offset);
+    if (offset != string::npos) {
+        int status = SToInt(methodLine.substr(offset));
+        if (status) {
+            return status;
+        }
+    }
+
+    // Default case, return 400
+    return 400;
```

Comment:
Why 400? Is this what we already had?

Reply:
Yes, that was copied from the comment from the original implementation here: https://github.com/Expensify/Server-Expensify/blob/30fdf559dd524d014b59e172555699b4ce584d36/auth/lib/Stripe.cpp#L70-L88

Reply:
If only anyone knew why we chose that value...